ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -2756,6 +2756,85 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-openstack-externallb,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Could I ask the intent behind this job? Most PRs should be covered by the existing hypershift job + regular MCO jobs, so I don't think we would need to always_run this

Reply:
that's a mistake. Sorry, let me fix this